docs/developer/apidocs/sparse.rst

```diff
@@ -94,3 +94,16 @@ par_ilut
 gmres
 -----
 .. doxygenfunction:: gmres(KernelHandle* handle, AMatrix& A, BType& B, XType& X, Preconditioner<AMatrix>* precond)
+
+sptrsv
+------
+.. doxygenfunction:: sptrsv_symbolic(const ExecutionSpace &space, KernelHandle *handle, lno_row_view_t_ rowmap, lno_nnz_view_t_ entries)
+.. doxygenfunction:: sptrsv_symbolic(KernelHandle *handle, lno_row_view_t_ rowmap, lno_nnz_view_t_ entries)
+.. doxygenfunction:: sptrsv_symbolic(ExecutionSpace &space, KernelHandle *handle, lno_row_view_t_ rowmap, lno_nnz_view_t_ entries, scalar_nnz_view_t_ values)
```

Comment:
@e10harvey Why do we need two interfaces for execution space: one with const and one without?

I am not sure if you are aware of this: ```sptrsv_symbolic``` is currently running on host, I think adding execution space to ```sptrsv_symbolic``` would not be very helpful just for now.